eng/Versions.props

```diff
@@ -78,13 +78,14 @@
        - Feed URI in the nuget.config
        - Native assets build and sha
     -->
-    <_SkiaSharpVersion>2.88.4-preview.82</_SkiaSharpVersion>
-    <_HarfBuzzSharpVersion>2.8.2.4-preview.82</_HarfBuzzSharpVersion>
-    <_SkiaSharpNativeAssetsVersion>0.0.0-commit.936270ba9372ac6c94505de44884ef4ba5d96aa2.588</_SkiaSharpNativeAssetsVersion>
+    <_SkiaSharpVersion>2.88.6</_SkiaSharpVersion>
```

Comment:
@mattleibow  does this look good? i resolved the conflit not sure if its fine

Reply:
Looks all good!